data/InMemoryDataStore.test.js

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable no-plusplus */
```

Comment:
You can set overrides in eslint for specific files so you don't have to keep adding these to the test files. You can see an example [here](https://stackoverflow.com/a/55184215/1123241)